app/controllers/users_controller.rb

```diff
@@ -1589,6 +1603,64 @@ def register_second_factor_security_key
     render json: failed_json.merge(error: err.message)
   end
 
+  def create_passkey
```

Comment:
I'm not sure if `create_passkey` is the right name here considering that the passkey is never really created on our end but by the external provider. Also, I'm not sure if this needs to be a `POST` request since we don't actually end  up creating any resource on the server.

Reply:
Yeah, you're right. I'm mostly sticking to the same convention used by `create_second_factor_security_key` and `create_second_factor_totp` here, with both naming and the use of the POST verb.

Reply:
Ahh icic. That is fair enough :+1: